### ISSUE TITLE
Dependents | Fix header on confirmation page

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#107550]
## Issue Description

The Frontend CoP team will be updating Cypress Axe tests to;
- Ensure page headers aren't empty
- Check the appropriate order of headers on a page
- Ensure every page includes a header

A preliminary run with these new rules enabled indicated an issue with the [686C-674](https://github.com/department-of-veterans-affairs/vets-website/actions/runs/13770651994/job/38508963343) (v1, v2 and anciilliary)

After running the e2e test locally, it appears that the failure is now only in the v1 form on the confirmation page header level order - an `H3` inside the alert ("Form submission started on...")

## Acceptance Criteria
- [x] Fix header order level
- [x] Ensure E2E & axe tests are passing
- [x] Review all e2e test with new axe checks enabled

## Definition of Done
- [x] All tests passing
- [x] Axe checks all passing

**Comment by Mottie** [comment]
Ready for validation in staging.

**Comment by Midge-dev** [comment]
LGTM

![Image](https://github.com/user-attachments/assets/5f230de7-5e4b-4abb-bc4a-70a0947fa00e)
